src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -164,6 +162,41 @@ public void dateToCurrentPosition(CaseData caseData) {
         }
     }
 
+    public void amendRespondentNameRepresentativeNames(uk.gov.hmcts.ecm.common.model.ccd.CaseData caseData) {
```

Comment:
not a massive issue but why are we using the package name here when defining our arguments? same with below method. Especially when casedata is already imported